data/126/021/452/3/1260214523.geojson

```diff
@@ -6,21 +6,21 @@
     "edtf:inception":"uuuu",
     "geom:area":0.0,
     "geom:area_square_m":0.0,
-    "geom:bbox":"-10.2,11.73333,-10.2,11.73333",
+    "geom:bbox":"-10.18,11.73333,-10.18,11.73333",
     "geom:latitude":11.73333,
-    "geom:longitude":-10.2,
+    "geom:longitude":-10.18,
     "gn:admin1_code":"K",
     "gn:admin2_code":"07",
-    "gn:asciiname":"Tomba Dougoula",
+    "gn:asciiname":"Tomba kanssa",
```

Comment:
GeoNames (`gn` prefixed properties) shouldn't be modified, they preserve what GeoNames knew about the place when we imported it.